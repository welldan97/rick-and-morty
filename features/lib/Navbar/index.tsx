import { memo } from 'react';

const Navbar = memo(() => (
  <nav className="navbar shadow-lg bg-base-100 z-50 relative">
    <h1>Rick &amp; Morty</h1>
  </nav>
));

export default Navbar;
