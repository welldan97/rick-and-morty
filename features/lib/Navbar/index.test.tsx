import { render } from '@testing-library/react';

import Navbar from './index';

describe('Navbar', () => {
  test('render', () => {
    const { container } = render(<Navbar>{() => <>test</>}</Navbar>);

    expect(container).toMatchSnapshot();
  });
});
