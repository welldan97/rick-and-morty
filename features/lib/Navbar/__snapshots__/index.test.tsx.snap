// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar render 1`] = `
<div>
  <nav
    class="navbar shadow-lg bg-base-100 z-50 sticky top-0 h-48 -translate-y-32 pt-32 flex items-center py-0 w-full"
  >
    test
    <a
      href="/"
    >
      <img
        alt="Rick & Morty"
        class="h-10 mx-2"
        src="logo.png"
      />
    </a>
  </nav>
</div>
`;
